### ISSUE TITLE
Support RAML 1.0

### ISSUE THREAD

**Issue by LeonidRosin** [SmartBear/soapui-raml-plugin#29]
Hi,
Any news about RAML 1.0 support?
I'm very interested, but SoapUI 5.4.0 still does not support RAML 1.0

**Comment by IlyaAvdeev** [comment]
Hi, @LeonidRosin 
ReadyAPI 2.6 supports RAML 1.0. SoapUI 5.4 and the latest 5.5 do not.

**Comment by hellboy81** [comment]
Actually ReadyAPI 2.8 still use old `raml-parser` dependency which does not support RAML 1.0

**Comment by hellboy81** [comment]
>ReadyAPI 2.6 supports RAML 1.0

Wrong! `Update definition` in both 2.6 and 2.8 still use old RAML parser without 1.0 spec support
